pages/community/index.tsx

```diff
@@ -0,0 +1,400 @@
+                  py={{ base: 12, md: 24 }}
+                >
+                  <Heading
+                    as="h2"
+                    textAlign="left"
+                    size="title.2xl"
+                    color="#ECECEC"
+                  >
+                    {title1}{" "}
+                    <Heading
+                      as="span"
+                      size="title.2xl"
+                      color="transparent"
+                      background="linear-gradient(249.19deg, #BFA3DA 53.72%, #84309C 78.01%, #C735B0 112.02%), #ECECEC"
+                      backgroundClip="text"
+                    >
+                      {title2}
+                    </Heading>
+                  </Heading>
+
+                  <Text
+                    mt={2}
+                    textAlign="left"
+                    size="body.xl"
+                    color="white"
+                    opacity={0.7}
+                    maxW="container.md"
+                  >
+                    {description}
+                  </Text>
+
+                  <SimpleGrid
+                    columns={{ base: 2, sm: 3, md: 4 }}
+                    gap={{ base: 12, md: 6 }}
+                    py={{ base: 12, md: 8 }}
+                  >
+                    {cards.map(
+                      ({
+                        image,
+                        title: cardTitle,
+                        link,
+                        description: cardDescription,
+                      }) => (
+                        <CommunityCard
+                          key={cardTitle}
+                          image={image}
+                          title={cardTitle}
+                          link={link}
+                          description={cardDescription}
+                        />
+                      ),
+                    )}
+                  </SimpleGrid>
+                </Container>
+              </Box>
+            ),
+          )}
+
+          <Box
+            h="1px"
+            bg="#3F3F3F"
+            opacity="0.8"
+            maxW="container.page"
+            mx="auto"
+          />
+        </Box>
+
+        <Container maxW="container.page" position="relative" py={12}>
+          <Heading as="h2" size="title.2xl" textAlign="center">
+            and most importantly...{" "}
+            <Heading
+              as="span"
+              size="title.2xl"
+              color="transparent"
+              background="linear-gradient(249.19deg, #BFA3DA 53.72%, #84309C 78.01%, #C735B0 112.02%), #ECECEC"
+              backgroundClip="text"
+            >
+              have fun!
+            </Heading>
+          </Heading>
+
+          <SimpleGrid
+            columns={{ base: 1, md: 3 }}
+            gap={{ base: 12, md: 6 }}
+            py={{ base: 12, md: 8 }}
+          >
+            {Array.from({ length: 9 }).map((_, i) => (
+              <ChakraNextImage
+                key={i}
+                src={`/assets/community/gallery/${i + 1}.png`}
```

Comment:
these should be `require(...)` so that next can optimize the images for us

Reply:
not sure what you mean here